build/QuickStatements/README.md

```diff
@@ -33,22 +33,14 @@ File                      | Description
 `/templates/oauth.ini`    | Template for Quickstatements' oauth.ini (substituted to `/var/www/html/quickstatements/oauth.ini` at runtime)                            
```

Comment:
Sorry, the below is somewhat noisy. There are not currently any textual changes, just removal of extraneous linebreaks within the paragraphs.